drivers/impuls-magnetic-mib.c

```diff
@@ -0,0 +1,281 @@
+/*  impuls_magnetic_mib.c - (ietf-mib.c + fix for upsload for IMPULSE SNMP-card for 1-10kVA UPS)  - data to monitor SNMP UPS (RFC 1628 compliant) with NUT
+ *
+ *  Copyright (C) 2002-2006
+ *  2012-2024   Sergei Kartavtsev <sergega@me.com>
```

Comment:
* I think the first set of years (2002-2006) with no name can go away
* Your own (C) is likely just 2024?
* Please align indentation with other lines